tests/components/justnimbus/test_config_flow.py

```diff
@@ -58,25 +57,7 @@ async def test_form_invalid_auth(hass: HomeAssistant) -> None:
         )
 
     assert result2["type"] == FlowResultType.FORM
-    assert result2["errors"] == {"base": "invalid_auth"}
-
-
-async def test_form_cannot_connect(hass: HomeAssistant) -> None:
-    """Test we handle cannot connect error."""
-    result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={"source": config_entries.SOURCE_USER}
-    )
+    assert result2["errors"] == errors
 
-    with patch(
-        "justnimbus.JustNimbusClient.get_data",
-        side_effect=JustNimbusError,
-    ):
-        result2 = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            {
-                CONF_CLIENT_ID: "test_id",
-            },
-        )
-
-    assert result2["type"] == FlowResultType.FORM
-    assert result2["errors"] == {"base": "cannot_connect"}
+    # check if it's still possible to configure the integration/no weird side-effects were caused
+    await test_form(hass=hass)
```

Comment:
This won't work, as it would create a new flow. Instead complete the flow right here with the current flows ID (next step)

Reply:
Should be better now, while still reusing the code